README.md

```diff
@@ -35,6 +35,29 @@ prostředí dají spustit pomocí:
 $ poetry run python -m pytest
 ```
 
+## Nasazení
+
+Aplikace jede na [rosti.cz](https://rosti.cz/). Pro nasazení je potřeba:
```

Comment:
It's great to have the instructions about the deployment, however I miss the information who to talk to about deployment. Is it only you or is there a bigger group of content curators? Should we mention it here?

Reply:
It's a bigger group but I don't know thde details. I opened https://github.com/pyvec/docs.pyvec.org/issues/288